packages/amplify_auth_cognito/example/test_driver/app.dart

```diff
@@ -9,3 +10,4 @@ void main() {
   // any widget you are interested in testing.
   app.main();
 }
+ */
```

Comment:
Do we need this file at all?

Reply:
@haverchuck what are your thoughts - can I delete these?

Reply:
Confirmed with Dustin that we can delete these.